.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*    @dialoguemd/python-guild
```

Comment:
> Unknown owner on line 1: make sure the team @dialoguemd/python-guild exists, is publicly visible, and has write access to the repository

Not sure if this will work

Reply:
I see, team is not publicIIUC. Ok, will remove for now, was not the goal of PR.